src/env_loader.zig

```diff
@@ -329,13 +329,24 @@ pub const Loader = struct {
     pub fn loadProcess(this: *Loader) void {
         if (this.did_load_process) return;
 
-        // This is a little weird because it's evidently stored line-by-line
-        var source = logger.Source.initPathString("process.env", "");
-
         this.map.map.ensureTotalCapacity(std.os.environ.len) catch unreachable;
-        for (std.os.environ) |env| {
-            source.contents = bun.span(env);
-            Parser.parse(&source, this.allocator, this.map, true, true);
+        for (std.os.environ) |_env| {
+            var env = bun.span(_env);
+            if (strings.indexOfChar(env, '=')) |i| {
```

Comment:
wonder what happens if an env key has an `=` in it

Reply:
according to Stack Overflow, this is correct

https://stackoverflow.com/questions/41109789/is-the-equals-sign-character-allowed-in-the-value-of-an-environment-value-in